README.md

```diff
@@ -27,7 +27,7 @@ Run `make` to build `./bin/firehose.so`. Then use with Fluent Bit:
 * `endpoint`: Specify a custom endpoint for the Kinesis Firehose API.
 * `sts_endpoint`: Specify a custom endpoint for the STS API; used to assume your custom role provided with `role_arn`.
 * `time_key`: Add the timestamp to the record under this key. By default the timestamp from Fluent Bit will not be added to records sent to Kinesis.
-* `time_key_format`: [strftime](http://man7.org/linux/man-pages/man3/strftime.3.html) compliant format string for the timestamp; for example, `%Y-%m-%dT%H:%M:%S%z`. This option is used with `time_key`. You can also use `%L` for milliseconds and `%f` for microseconds.
+* `time_key_format`: [strftime](http://man7.org/linux/man-pages/man3/strftime.3.html) compliant format string for the timestamp; for example, `%Y-%m-%dT%H:%M:%S%z`. This option is used with `time_key`. If you are running Amazon ECS Container Agent v42.0 or later, you can also use `%L` for milliseconds and `%f` for microseconds.
```

Comment:
Please use this wording, or something very similar: "You can also use `%L` for milliseconds and `%f` for microseconds. If you are using ECS FireLens, make sure you are running Amazon ECS Container Agent v42.0 or later, otherwise the timestamps associated with your container logs will only have second precision."

The current wording lacks enough context, remember that folks use this plugin who have no knowledge of ECS (because they're k8s users).

Also is it v42 or v1.42? Pretty sure its the latter.

Reply:
Good point 👍🏼